aten/src/ATen/Context.h

```diff
@@ -94,6 +94,8 @@ class CAFFE2_API Context {
 
   bool setFlushDenormal(bool on);
 
+  enum ErrorLevel { None=0, Warn, Error };
```

Comment:
I think I would recommend (1) making this an enum class, (2) naming it more descriptively so that it is specifically about determinism options, (3) moving it outside of Context class